Scripts: svg is no longer able to be loaded as described in the docs

### Description

As of `@wordpress/scripts` `32.2.0`, this technique for using SVG in a block no longer works and throws an error at compile/build time.

```js
import { ReactComponent as Explode } from './explode.svg';

return (
    <Explode />
);
```

This technique is described in [the Using SVG section](https://github.com/WordPress/gutenberg/tree/trunk/packages/scripts#using-svg) in the wp-scripts docs.

Instead of working, I get the following error:
ERROR in ./src/my-block/save.js 4:0-61
Module not found: Error: Can't resolve 'url-loader' in '/Users/jparis/projects/html/wp-content/plugins/my-plugin'


### Step-by-step reproduction instructions

1. in your block main directory (next to block.json) add a svg file.
2. install `@wordpress/scripts@^32.2`
3. In your edit.js and/or save.js, add the code :
    ```
    import { ReactComponent as Explode } from './explode.svg';
      
      return (
          <Explode />
      );
    ```
4. run `wp-scripts build` (or `wp-scripts start`). 
5. see the error

### Screenshots, screen recording, code snippet

_No response_

### Environment info

- WordPress 7.0
- @wordpress/scripts 32.2.0

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [x] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

--- a/packages/scripts/config/webpack.config.js
+++ b/packages/scripts/config/webpack.config.js
@@ -1,17 +1,17 @@
 /**
  * External dependencies
  */
+const { basename, dirname, relative, resolve, sep } = require( 'path' );
+const { realpathSync } = require( 'fs' );
+const { exec } = require( 'child_process' );
 const { BundleAnalyzerPlugin } = require( 'webpack-bundle-analyzer' );
 const CopyWebpackPlugin = require( 'copy-webpack-plugin' );
 const webpack = require( 'webpack' );
 const browserslist = require( 'browserslist' );
 const MiniCSSExtractPlugin = require( 'mini-css-extract-plugin' );
-const { basename, dirname, relative, resolve, sep } = require( 'path' );
 const ReactRefreshWebpackPlugin = require( '@pmmmwh/react-refresh-webpack-plugin' );
 const TerserPlugin = require( 'terser-webpack-plugin' );
-const { realpathSync } = require( 'fs' );
 const { sync: glob } = require( 'fast-glob' );
-const { exec } = require( 'child_process' );
 
 /**
  * WordPress dependencies
@@ -220,7 +220,10 @@ const baseConfig = {
 			{
 				test: /\.svg$/,
 				issuer: /\.(j|t)sx?$/,
-				use: [ '@svgr/webpack', 'url-loader' ],
+				use: [
+					require.resolve( '@svgr/webpack' ),
+					require.resolve( 'url-loader' ),
+				],
 				type: 'javascript/auto',
 			},
 			{

